content/en/docs/polo-features/operations-panel.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Operations Panel
+weight: 2
+description: How PoLo is operated.
+
+---
+
+## Shortcut Polo Features
```

Comment:
what does this subtitle mean?

Reply:
A misunderstanding of how these shortcuts are set to work. I thought it might take the user up a level within the doc tree.